### PR TITLE
Gives Detective ACCESS_SECURITY, moves security equipment to ACCESS_BRIG

### PR DESCRIPTION
**Pull request by Son-of-Space**
## About The Pull Request

This PR gives the detective ACCESS_SECURITY by default, following the general standards set up for other departmental access where jobs in a department get general access to their department.

I moved access to the gear room to ACCESS_BRIG, removed the detective's access to that at roundstart, and added ACCESS_BRIG to their extra access.

I updated the comments on access.dm to properly reflect this and updated the secure storage closets to still allow detectives access. (We do this so if someone gets deputized they can access equipment without potentially accessing case material or antag gear.)

I also fixed a door on Delta that was still using access varedits rather than helpers in the Det's office but that's something I should've fixed in the first iteration. ; )
## Why It's Good For The Game

Having general access under all job functions makes it easier to prevent situations where we need to give niche access to certain items so the detective can have it, making it harder to miss adding this access, etc. Areas such as the evidence lockers, office, firing range, and interrogation no longer need more than one access for every intended role to enter.

Having specialized equipment under a specialized role makes it easier for us to properly lock it away from roles that should not have it. For example, detectives are not generally expected to brig, gulag, or otherwise imprison others, but because this access was not simplified, they had it at roundstart despite not having access to the equipment normally used to imprison crew. Now access fits the logical distinction of the roles while still enabling them to perform those functions on a skeleton crew.

## Changelog

:cl:
qol: Security General access has been simplified
fix: Detectives ability to brig crew has been moved to skeleton crew access to fit the job function
fix: Detectives now have general access to all general areas of security consistently across all stations
/:cl:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -176,7 +176,7 @@
 
 /obj/structure/closet/secure_closet/brig
 	name = "brig locker"
-	req_access = list(ACCESS_BRIG)
```
Comment (JohnFulpWillard) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:179]: Why not security access here?

**Review by JohnFulpWillard — CHANGES_REQUESTED** [review]
Sec records console access should remove forensics since detectives will have access to it now anyways

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 1 above)
Comment (Son-of-Space) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:179]: I believe this is the locker that officers use to secure prisoner's items, so it belongs to ACCESS_BRIG .

Comment (JohnFulpWillard) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:179]: But you’re also giving detective the access?

Comment (Son-of-Space) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:179]: Oh I think I see what you mean, I added forensics here when I don't believe I should've. I'll remove it.

**Comment by Palpetinus** [comment]
Finally. I got tired of having to roundstart ask HoP to give me "Security (Common)" and half of them not knowing what I mean.
By the way, that's the same access you need for SecHUD record editing, meaning Dets can roundstart fully use SecHUDs now. No idea if this was intended or if it matters, just felt like it deserved a mention.
(feels like they should have been able to from the start, given that they can already edit records on the console)

**Comment by JohnFulpWillard** [comment]
The only reason I haven't merged this yet is because I don't really like detectives having access to security lockers, even if its on lowpop. We can change it in the future if it's a problem though.

**Pull request merged**